csharp/msbuild/coverage.runsettings

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="Code Coverage" uri="datacollector://Microsoft/CodeCoverage/2.0"
+        assemblyQualifiedName="Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+        <Configuration>
+          <CodeCoverage>
+            <Sources>
+              <Include>
+                <Source>.*/src/.*</Source>
+              </Include>
+              <Exclude>
+                <Source>.*/generated/.*</Source>
```

Comment:
Can we keep this to catch unused, and untested generated code?

Reply:
I think looking the coverage for generated code in general just messes up the numbers and doesn't generally give any meaningful information. I think generally, most generated code is unused. We filter out generated code for IceRPC coverage as well.